assets/queries/cloudFormation/iam_password_has_uppercase_letter/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "IAM_password_has_uppercase_letter",
+  "queryName": "IAM Password Has Uppercase Letter",
```

Comment:
The query name should reflect the vulnerability, not the fix. We should use something like the reverse of what it's now.